modules/ROOT/pages/tokenization.adoc

```diff
@@ -2,16 +2,33 @@
 //You can determine when to apply this token via a lightweight API policy.
-You can determine when to apply this token using a lightweight API through a configured Tokenization Service.
 
-The tokenization service addresses a variety of data security needs:
-
-== Format Preserving Tokenization
-
-Application data validation logic work “as is”. +
-No downstream application changes needed. Compliance scope reduction (PCI, HIPAA, GDPR)
-
-image::index-a21f8.png[index-a21f8]
+The Anypoint Security tokenization service enables you to substitute a sensitive data element with a non-sensitive equivalent. Anypoint tokenization can create Format Preserving Tokens, which means the output tokens will have the same format as the sensitive data input.
+
+Examples of sensitive information that are suitable for tokenization protection include:
+
+* Primary Account Number (PAN)
+* Personally Identifiable Information (PII)
+* Protected Health Information (PHI), or any information deemed sensitive.
+
+Anypoint Security tokenization provides the following benefits:
+
+* A vaultless operation, ensuring sensitive data is not stored in the tokenization deployment:
+** No data growth based on the number of transactions processed or tokens issued, thus removing data-vault management issues.
+* Increased security for sensitive information thereby reducing your “Sensitive Data Zones”:
+** Reduces or removes the number of systems and applications that process, store, or transmit sensitive information from sensitive data scopes.
+** Negates the chances of a data breach from auxiliary applications and data stores, as the sensitive information stored in them is replaced by tokens.
+** Increases defense in depth for sensitive data and least privilege. Authorization is required to the tokenization service to detokenize the sensitive data back to its actual value. Possession of the token alone provides no extrinsic or exploitable meaning or value.
+* Flexibility:
+** Ability to choose a tokenization “scheme” appropriate for your business using inbuilt data domains (PAN, SSN, email, decimal, ascii and so on).
+** Each domain has “configurable” options for preserving portions of the original sensitive data and replacing the rest with token data or replacing the entire information with a token.
+** Operates seamlessly across wide area networks and distributed data centers.
+* Format Preserving Tokenization capability ensures that:
+** No changes are required to existing enterprise data flows or data stores, as the solution ensures that the tokens generated conform to the existing data structure and validations.
+** Changes to existing applications are minimized compared to point-to-point encryption, which requires data protection implementation at each “hop”.
+* Performance:
+** High-performance operation ensures low-latency processing.
+** Performance scales linearly on each node with vertical and horizontal scaling for vaultless tokenization provided by the Anypoint Platform.
```

Comment:
I don't understand how two-dimensional scaling can be linear scaling, and I don't really understand what this sentence means. do customers also have valutless tokenization not provided by Anypoint Platform, or is that an unnecessary qualifier? Each node of what? What are the vertical and horizontal axes being referenced?